components/general/fonts/_fonts.scss

```diff
@@ -1 +1,16 @@
-// this file probably shouldn't be empty...
+.font {
```

Comment:
I'm not sure about the use case of this component.  IMO it should just load font files, b/c using a component to set just `font-weight` is a misuse.

PS. All components in `general` are designed to be loaded once as a base or be a source for others, this one doesn't fit that rules.